modular_skyrat/master_files/code/modules/client/preferences/headshot.dm

```diff
@@ -0,0 +1,37 @@
+/datum/preference/text/headshot
+	category = PREFERENCE_CATEGORY_NON_CONTEXTUAL
+	savefile_identifier = PREFERENCE_CHARACTER
+	savefile_key = "headshot"
+	/// Stored headshot link from `value` to cut down on chat spam
+	var/stored_link
```

Comment:
this is a global datum used by literally every client, the code will literally not function for more than one player

Reply:
Made into an assoc list based on usr's ckey